kentik_api_library/kentik_api/synthetics/synth_tests/base.py

```diff
@@ -290,6 +291,46 @@ class SynTest(_ConfigElement):
     _created_by: UserInfo = field(default_factory=UserInfo, init=False)
     _last_updated_by: UserInfo = field(default_factory=UserInfo, init=False)
 
+    def _common_attribute_fixups(self):
+        # Correct settings.health_settings.activation parameters if necessary
+        orig = None
+        if not self.settings.health_settings.activation.times:
+            self.settings.health_settings.activation.times = "3"
+        min_activation_window = int(
+            self.settings.period * (int(self.settings.health_settings.activation.times) + 1) / 60
+        )
+        if not self.settings.health_settings.activation.time_window:
+            orig = "{}{}".format(
+                self.settings.health_settings.activation.time_window, self.settings.health_settings.activation.time_unit
+            )
+            self.settings.health_settings.activation.time_window = str(min_activation_window)
+            self.settings.health_settings.activation.time_unit = "m"
+        else:
+            # Fixup alert time window
+            # Convert current activation window to minutes
+            if self.settings.health_settings.activation.time_unit == "h":
```

Comment:
This change needs also update in community_sdk_python/kentik_api_library/tests/e2e/synthetics/utils.py:50, where time_unit "h" is used

Reply:
I'll look into it.

Reply:
Addressed in this commit https://github.com/kentik/community_sdk_python/pull/123/commits/aaec320e844653588e6cdf15b98404f4dfbbe8dc